libvast/src/format/syslog.cpp

```diff
@@ -12,9 +12,85 @@
  ******************************************************************************/
 
 #include "vast/format/syslog.hpp"
+#include "vast/error.hpp"
 
-namespace vast::format::mrt {
+namespace vast {
+namespace format {
+namespace syslog {
 
-// TODO: implement functions here that do not belong into the header file. 
+syslog_parser::syslog_parser() {
+  // Syslog header type.
+  auto header_fields = std::vector<record_field>{
+    {"facility", count_type{}},
+    {"severity", count_type{}},
+    {"time", time_type{}.attributes({{"timestamp"}})},
```

Comment:
```suggestion
    {"ts", time_type{}.attributes({{"timestamp"}})},
```

We usually name this field `ts`.